backend-api/src/functions/asyncActiveSession/tokenService/tokenVerifier.ts

```diff
@@ -0,0 +1,28 @@
+import { jwtVerify, JWTVerifyResult, KeyLike } from "jose";
+import { errorResult, Result, successResult } from "../../utils/result";
+
+export class TokenVerifier implements ITokenVerifier {
+  async verify(
+    jwt: string,
+    key: KeyLike | Uint8Array,
+  ): Promise<Result<JWTVerifyResult>> {
+    let result: JWTVerifyResult;
+    try {
+      result = await jwtVerify(jwt, key);
+    } catch (error) {
```

Comment:
As much as I don't like saying this, can we not have `(error)` and then just return `errorMessage: "Error verifying token signature"...?

Reply:
Yes - removed